apis/v1beta1/config.go

```diff
@@ -513,6 +565,32 @@ type MetricsConfig struct {
 
 	// Address is the [address]:port that metrics exposition should be bound to.
 	Address string `json:"address,omitempty" yaml:"address,omitempty"`
+
+	// Readers is the list of metric readers.
+	Readers []MetricReader `json:"readers,omitempty" yaml:"readers,omitempty"`
```

Comment:
i don't think this approach will work if we want to support users that set readers other than the prometheus one, can we not import the config package directly?

Reply:
> i don't think this approach will work if we want to support users that set readers other than the prometheus one, 

Why?

> can we not import the config package directly?

Why don't we do that for the full configuration we have here?

Reply:
if a user [sets this](https://github.com/open-telemetry/opentelemetry-configuration/blob/2e76e05faf862650b23363d2a0f4d6b34df9e646/examples/kitchen-sink.yaml#L259-L275), won't we just drop the config?

Reply:
I see.